accept/src/botReview.ts

```diff
@@ -2,10 +2,56 @@ import { getInput, info, warning } from '@actions/core'
 import { getOctokit } from '@actions/github'
 import { getPR, getShaFromBranch } from '../../lib/github'
 import { detectIssueKey } from './detectIssueKey'
+import { getEventData } from '../../lib/event'
 
 // id of exivity bot
 const EXIVITY_BOT = 53756225
 
+interface IDObject {
+  id: number
+}
+
+async function checkIfReady(
+  octokit: ReturnType<typeof getOctokit>,
+  ref: string,
+  repo: string
+): Promise<boolean> {
+  const event = process.env['GITHUB_EVENT_NAME']
+
+  if (event === 'push') return true
```

Comment:
I'd maybe explicitly check here for branch name === 'develop' || 'master', just to be sure